graphql/authorizations.ts

```diff
@@ -240,7 +240,8 @@ export const authorizationModelEnhanceMap: ModelsEnhanceMap = {
             state: adminOrOwner,
             teacherEmailAddress: adminOrOwner,
             coduToken: adminOrOwner,
-
+            lastTimeCheckedNotifications: adminOrOwner,
```

Comment:
Why is this here as well? 
Something off with the history?

Reply:
one is for Pupil and another for Student :)